definition_of_done.md

```diff
@@ -0,0 +1,37 @@
+# Definition of done
+
+This document clarifies, when a product backlog item is done. It will act as a checklist during the scrum development process.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api function descriptions (sphinx/doxygen are [autogenerated online](http://docs.ros.org/melodic/api/pilz_robots/html/))
+* [ ] API diagrams
+* [ ] [Tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
```

Comment:
```suggestion
* [ ] Update [tutorials](https://wiki.ros.org/pilz_robots/Tutorials/)
```

Reply:
"Up-to-date" could be added for all checklist items? Why is is more important here?